requirements.in

```diff
@@ -0,0 +1,12 @@
+amclient==1.1.1
+asterism==0.7.2
+Django==3.2.4
+djangorestframework==3.12.4
+health-check==3.4.1
+jsonschema==3.2.0
+psycopg2-binary==2.8.6
+pyfc4==0.6
+pylzma==0.5.0
```

Comment:
Is this needed? Can't remember if it's used for Django Rest Framework or no...

Reply:
Do you mean `pylzma`? Part of it is being imported and used `helpers.py`.

We're importing `py7zlib` which doesn't seem to have a separate import.